### PR TITLE
Better fix for flapping test - TestJetStreamClusterStaleReadsOnRestart

### PR DESCRIPTION
**Pull request by derekcollison**
Signed-off-by: Derek Collison <derek@nats.io>

### CODE REVIEW

**Review by neilalexander — APPROVED** [review]
LGTM

File: server/jetstream_cluster_2_test.go [hunk 1]
```diff
@@ -7095,25 +7095,31 @@ func TestJetStreamClusterStaleReadsOnRestart(t *testing.T) {
...
+		for _, s := range c.servers {
+			if s.Running() {
+				mset, err := s.GlobalAccount().lookupStream("TEST")
+				if err != nil {
+					return err
+				}
+				var fs StreamState
+				mset.store.FastState(&fs)
+				if !reflect.DeepEqual(fs, state) {
+					return fmt.Errorf("States do not match, exepected %+v but got %+v", state, fs)
```
Comment (wallyqs) [inline server/jetstream_cluster_2_test.go:7117]:
```suggestion
					return fmt.Errorf("States do not match, expected %+v but got %+v", state, fs)
```

**Review by wallyqs — APPROVED** [review]
LGTM

**Review by wallyqs — APPROVED** [review]
LGTM

File: server/jetstream_cluster_2_test.go (hunk 1 above)
Comment (derekcollison) [reply server/jetstream_cluster_2_test.go:7117]: Will fix.

**Pull request merged**